Integrating with your webapp/README.md

```diff
@@ -0,0 +1,167 @@
+# Integrate Anbox Cloud in your web application
+
+The dashboard included in Anbox Cloud is great to get started but writing your own web application
+
+## 0. AMC
+
+`amc` is the main tool you'll be using to interact with Anbox Cloud.
+
+ > `amc` is a simple wrapper over `ams` HTTP API. Everything you can do with `amc` can also be
+ > done through the API 
+
+### Accessing AMC
+
+Depending on how you deployed Anbox Cloud, `amc` can be accessed in different ways
+
+**Anbox Cloud Appliance (AWS Marketplace):**
+```sh
+$ ssh ubuntu@<instance IP>
+$ amc help
+```
+
+**Juju based deployment:**
+```sh
+$ juju ssh ams/0
+$ amc help
+```
+
+## 1. Creating the application
+
+Create a directory and place a file named `manifest.yaml` in it.
```

Comment:
I feel like it's more intuitive if we use heredoc rather than touch command.  In this way,things can be simplified with
```
    Create a directory and place a file named `manifest.yaml` in it with the following content.
    $ mkdir my-android10-app
    $ cat << EOF > manifest.yaml
      name: my-android10-app
      instance-type: a2.3
      EOF
```

Reply:
+1